admin/formatter/class-metabox-formatter.php

```diff
@@ -82,12 +82,28 @@ private function get_defaults() {
 	}
 
 	/**
+	 * Gets the locale used for translations. 
+	 *
+	 * @return mixed
+	 */
+	private function get_translation_locale() {
+		$locale = get_locale();
+
+		if ( function_exists ( 'get_user_locale' ) ) {
+			$locale = get_user_locale();
```

Comment:
I would choose for a direct return, to prevent an unnecessary function call to `get_locale`. See the example I have given. If you disagree with that I would like to hear why.